### PR TITLE
Fixed wrong template returned in list element context and service definition duplication

### LINKED ISSUE #50: remove map-builder service from services.xml
remove [map-builder](https://github.com/fsi-open/admin-translatable-bundle/blob/master/Resources/config/services.xml#L16) as it [is defined already](https://github.com/fsi-open/admin-translatable-bundle/blob/master/Resources/config/context/resource.xml#L27)

### PR DESCRIPTION
**Pull request by szymach**
Resolves #50

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -12,13 +12,6 @@
     </parameters>
 
     <services>
-        <!--Map Builder-->
-        <service id="admin_translatable.resource.map_builder" class="%admin_translatable.resource.map_builder.class%">
```
Comment (chives) [inline Resources/config/services.xml]: @szymach are you sure that this definition is useless?

Comment (szymach) [inline Resources/config/services.xml]: Yes, it is repeated [here](https://github.com/fsi-open/admin-translatable-bundle/blob/master/Resources/config/context/resource.xml#L23) and having it here makes the whole bundle dependent on `ResourceRepository`, always

Comment (szymach) [inline Resources/config/services.xml]: I need to fix it

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@rn0

**Pull request merged**